frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -600,9 +600,8 @@ const ResolvedFields& fieldsForTypeDeclQuery(Context* context,
   bool isMissingBundledType = false;
   if (auto bct = ct->toBasicClassType()) {
     isObjectType = bct->isObjectType();
-    auto id = bct->id();
-    isMissingBundledType = CompositeType::isMissingBundledType(context, id);
   }
+  isMissingBundledType = CompositeType::isMissingBundledType(context, ct->id());
```

Comment:
could this assignment be moved to the declaration? It doesn't seem like there's a need to separate them here anymore